google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadOfflineConversions.java

```diff
@@ -0,0 +1,162 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+package com.google.ads.googleads.examples.remarketing;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v1.utils.ResourceNames;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.services.ClickConversion;
+import com.google.ads.googleads.v2.services.ClickConversionResult;
+import com.google.ads.googleads.v2.services.ConversionUploadServiceClient;
+import com.google.ads.googleads.v2.services.UploadClickConversionsResponse;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.DoubleValue;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/** Imports offline conversion values for specific clicks to an account. */
+public class UploadOfflineConversions {
+  private static class UploadOfflineConversionsParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private long customerId;
+
+    @Parameter(names = ArgumentNames.CONVERSION_ACTION_ID, required = true)
+    private long conversionActionId;
+
+    @Parameter(names = ArgumentNames.GCL_ID, required = true)
+    private String gclid;
+
+    @Parameter(names = ArgumentNames.CONVERSION_TIME, required = true)
```

Comment:
Please add a description that explains the expected format of this parameter

Reply:
Description looks good, but could you add it to the `description` attribute of the `@Parameter` annotation instead? That way, users will see it on the console when they run this example with `--help`.